client/src/common/MomentumClient.js

```diff
@@ -58,6 +53,13 @@ const deleteProgramme = (id) => {
     });
 };
 
+const updateProgramme = (id, programme) => {
+    return ApiClient(`programmedegree/${id}`, {
```

Comment:
Programmedegree no longer requires the id to make it the same as other PUT methods. It should ping `programmedegree`